sky/clouds/kubernetes.py

```diff
@@ -394,3 +395,22 @@ def query_env_vars(cls, name: str) -> Dict[str, str]:
                 env_vars[key] = line[1]
 
         return env_vars
+
+    @classmethod
+    def get_current_user_identity(cls) -> Optional[List[str]]:
+        k8s = kubernetes.get_kubernetes()
+        try:
+            _, current_context = k8s.config.list_kube_config_contexts()
+            if 'namespace' in current_context['context']:
+                namespace = current_context['context']['namespace']
+            else:
+                namespace = kubernetes_utils.DEFAULT_NAMESPACE
+
+            user = current_context['context']['user']
+            cluster = current_context['context']['cluster']
+            return [
+                hashlib.md5(
+                    f'{cluster}_{user}_{namespace}'.encode()).hexdigest()
```

Comment:
Instead of the md5 hash, should we simply use the `f'{cluster}_{user}_{namespace}'` string as the hash?

That way, the user gets a human-readable message when they try to down a cluster different from their current context:

```
# Current: 
(base) ➜  ~ sky down test
Terminating 1 cluster: test. Proceed? [Y/n]: Y
'test' (Kubernetes) is owned by account ['dff0567c57c004f11932d493103aa309'], but the activated account is ['64cdb455843c42f5ee4d6d1dbe5e7fc6'].

# Would be easier to debug for the user:
(base) ➜  ~ sky down test
Terminating 1 cluster: test. Proceed? [Y/n]: Y
'test' (Kubernetes) is owned by account ['kind-skypilot_kind-skypilot_default'], but the activated account is ['gke_skypilot-375900_us-central1-c_testclustergpu_gke_skypilot-375900_us-central1-c_testclustergpu_default'].
```

Reply:
Updated in https://github.com/skypilot-org/skypilot/pull/2522/commits/46cc5920bfe1a235916cbe663e61c7aa9cda8f3f